FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -2506,7 +2506,17 @@ def installDatabase( self, dbName ):
 
     gLogger.notice( 'Installing', dbName )
 
-    dbFile = glob.glob( os.path.join( rootPath, '*', '*', 'DB', '%s.sql' % dbName ) )
+    dbFile = glob.glob(os.path.join( rootPath, 'DIRAC', '*', 'DB', '%s.sql' % dbName ))
```

Comment:
shouldn't this be mixed somehow with the generic "object loader" we have ? 
it is the same principle after all.

Reply:
It most probably is. I don't know why in this module is not used, but right now I am not keen in changing.